LogMonitor/LogMonitorTests/LogMonitorTests.cpp

```diff
@@ -5,7 +5,6 @@
 
 #include "pch.h"
 
-#include "../src/LogMonitor/ConfigFileParser.cpp"

```

Comment:
Do we not need this anymore?

Reply:
Thanks for catching this. It's not needed anymore. Removing it.